inginious/frontend/pages/course_admin/settings.py

```diff
@@ -76,6 +76,7 @@ def POST_AUTH(self, courseid):  # pylint: disable=arguments-differ
                 errors.append(_('Invalid ACL value'))
             if course_content['registration_ac'] == "None":
                 course_content['registration_ac'] = None
+            course_content['registration_ac_accept'] = data['registration_ac_accept']
```

Comment:
You don't store a boolean here, but you load the value and use it as a boolean in the remaining of the code.

Reply:
Fixed in fb574ef83152d1534d3f41a08fbe21415547fcdc.